frontend/packages/dev-console/src/components/import/DeployImage.tsx

```diff
@@ -69,6 +69,7 @@ const DeployImage: React.FC<Props> = ({
       status: { metadata: {}, status: '' },
       ports: [],
     },
+    runtimeIcon: null,
```

Comment:
```suggestion
    runtimeIcon: 'openshift',
```
Design says the default is openshift.

Reply:
The code defines now openshift as default in the IconDropdown, so I would prefer to just keep the default there and start with null, same as editing a Deployment which does not have a runtime label yet.